src/aat/resources/features/F-037/F-037.feature

```diff
@@ -66,3 +66,24 @@ Feature: F-037: Submit event for an existing case (V2)
     Then a negative response is received
     And the response [contains a HTTP 404 'Not Found']
     And the response has all the details as expected
+
+  @S-577
+  Scenario: must update successfully all and only the respective fields with update access for a Case Worker and a Solicitor
+    Given a user with [an active Case Worker profile in CCD with a specific variation of ACLs on a case type]
+    And a user with [an active solicitor profile with another specific variation of ACLs on the same case type]
```

Comment:
we could mention the privileged user with full permissions here as well

Reply:
Yes, make sense. updated.

Reply:
I meant in addition to the other solicitors. We are using three different solicitors in this scenario so there should be a mention of all three here

Reply:
ohk, I have added the three entry now.